apollo-ios/Sources/Apollo/NetworkFetchInterceptor.swift

```diff
@@ -36,7 +36,11 @@ public class NetworkFetchInterceptor: ApolloInterceptor, Cancellable {
       return
     }
     
-    let task = self.client.sendRequest(urlRequest) { [weak self] result in
+    var taskDescription = "\(Operation.operationType) \(Operation.operationName)"
+    if let operationIdentifier = Operation.operationIdentifier {
```

Comment:
These operation identifiers are just the id's used for persisted queries. I don't think they are really useful in this context.

Reply:
Sounds good to me. I removed it in commit 897be2f